idl/spartan7-loader.idol

```diff
@@ -0,0 +1,13 @@
+// Interface to Spartan-7 FPGA loader.
+
+Interface(
+    name: "Spartan7Loader",
+    ops: {
+        "ping": (
```

Comment:
take it or leave it: i might have named this something more descriptive, like "wait_for_completion" or something...?

Reply:
I'd prefer to stick with `ping` – the operation is so fundamentally basic that I don't want the op name to imply any complexity.

Reply:
reasonable, carry on!